weeve-agent-installer.sh

```diff
@@ -22,91 +25,114 @@ cleanup() {
   fi
 }
 
-process_complete=false
+PROCESS_COMPLETE=false
 
 log Read command line arguments ...
 
-key=$(echo "$@" | cut --fields 1 --delimiter='=')
-value=$(echo "$@" | cut --fields 2 --delimiter='=')
+KEY=$(echo "$@" | cut --fields 1 --delimiter='=')
+VALUE=$(echo "$@" | cut --fields 2 --delimiter='=')
 
-if [ "$key" = NodeName ] ; then
-  node_name="$value"
+if [ "$KEY" = NodeName ] ; then
+  NODE_NAME="$VALUE"
 fi
 
-if [ -z "$node_name" ]; then
-log node_name is required. | Argument Name: NodeName
-exit 0
+if [ -z "$NODE_NAME" ]; then
+log NODE_NAME is required
+read -p "Give a node name: " NODE_NAME
 fi
 
 log All arguments are set
-log Name of the node: "$node_name"
+log Name of the node: "$NODE_NAME"
+
+log Checking if a instance of weeve-agent is already running ...
+
+CURRENT_DIRECTORY=$(pwd)
+
+WEEVE_AGENT_DIRECTORY="$CURRENT_DIRECTORY"/weeve-agent
+SERVICE_FILE=/lib/systemd/system/weeve-agent.service
+ARGUMENTS_FILE=/lib/systemd/system/weeve-agent.argconf
 
-secret_file=.weeve-agent-secret
+if [ -d "$WEEVE_AGENT_DIRECTORY" ] || [ -f "$SERVICE_FILE" ] || [ -f "$ARGUMENTS_FILE" ]; then
+  log Detected some weeve-agent contents!
```

Comment:
It's better to let user decide, prompt user message that weeveAgent is already installed and ask if they want to continue or cancel installation (providing some options), just like we see while installing any application in Linux using command prompt.
Follow the same at all places as needed